src/Modules/Dashboard/DashBoardDataComponent.js

```diff
@@ -1,35 +1,40 @@
 import React from 'react';
-import { withRouter } from 'react-router-dom';
+import {withRouter} from 'react-router-dom';
 import SearchComponent from '../General/SearchComponent';
 import DashboardCreateAccountComponent from './DashboardCreateAccountComponent';
-import '../../Assets/scss/main.scss';
+import { useForm } from 'react-hook-form';
+
 const DashBoardDataComponent = (props) => {
-  // Login is out of scope
-  // const [isLoggedIn, setIsLoggedIn] = React.useState(localStorage.getItem('isLoggedIn'));
+    // Login is out of scope
```

Comment:
Non-blocking but can we standardize 2 spaces vs tabs? Makes diffs hard to read.

Reply:
Sure. Which one do you prefer?